build.gradle.kts

```diff
@@ -1,19 +1,3 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        google()
-        gradlePluginPortal()
-    }
-    dependencies {
-        classpath(libs.com.android.tools.build.gradle)
-    }
-}
-
-repositories {
```

Comment:
Now declared in [settings.gradle.kts / dependencyResolutionManagement](https://github.com/KakaoCup/Kakao/pull/109/files#diff-5625e3601fa0ad3a6a2824239e5a2fde71c149597d31394f9224a08c24be7b9dR9)